app/assets/javascripts/discourse/app/mixins/uppy-upload.js

```diff
@@ -112,7 +123,9 @@ export default Mixin.create(UppyS3Multipart, ExtendableUploader, {
           },
           this.validateUploadedFilesOptions()
         );
-        const isValid = validateUploadedFile(currentFile, validationOpts);
+        const isValid =
+          validateUploadedFile(currentFile, validationOpts) &&
+          this.validateUploadedFile(currentFile);
```

Comment:
Can one of those be renamed? I spent like a minute starring at it trying to figure out why the same function is called twice, with different args 😵‍💫 😅

Reply:
Sure thing, I'm just undecided about a better name... ideas? I can rename the class method to something like `isUploadedFileValid` but it still feels a bit ambiguous.

Maybe using "allow"/"allowed" in the overridable method? `isUploadedFileAllowed` or `allowUploadedFile`, maybe. WDYT?